src/status_im/contexts/centralized_metrics/tracking.cljs

```diff
@@ -41,11 +44,16 @@
 
 (defn track-view-id-event
   [view-id]
-  (if-let [screen (get screens/screens-by-name view-id)]
-    (when (get-in screen [:metrics :track?])
-      (screen-event screen {}))
-    (when (contains? view-ids-to-track view-id)
-      (navigation-event (name view-id)))))
+  (let [screen (get screens/screens-by-name view-id)]
+    (cond-> []
+      (get-in screen [:metrics :track?])
+      (conj (screen-event screen {}))
+
+      (contains? view-ids-to-track view-id)
+      (conj (navigation-event (name view-id)))
+
+      (= :screen/onboarding.syncing-results view-id)
```

Comment:
FYI: The idea is that when the `view-id` is equal to `:screen/onboarding.syncing-results` happen we want to publish two events, the normal one and the new `onboarding-completed` event. In the interceptor `centralized-metrics.events/centralized-metrics-interceptor` we process an event as a collection, that's why this works:

```clojure
(doseq [e event]
  (native-module/add-centralized-metric e))
```